databricks/koalas/config.py

```diff
@@ -298,3 +298,53 @@ def _check_option(key: str) -> None:
         raise OptionError(
             "No such option: '{}'. Available options are [{}]".format(
                 key, ", ".join(list(_options_dict.keys()))))
+
+
+class DictWrapper:
+    """ provide attribute-style access to a nested dict"""
+
+    def __init__(self, d, prefix=""):
+        object.__setattr__(self, "d", d)
+        object.__setattr__(self, "prefix", prefix)
+
+    def __setattr__(self, key, val):
+        prefix = object.__getattribute__(self, "prefix")
+        d = object.__getattribute__(self, "d")
+        if prefix:
+            prefix += "."
+        canonical_key = prefix + key
+
+        candidates = [
+            k for k in d.keys() if all(x in k.split(".") for x in canonical_key.split("."))]
+        if len(candidates) == 1 and candidates[0] == canonical_key:
+            return set_option(canonical_key, val)
+        elif len(candidates) == 0:
```

Comment:
We don't need to check here but just raise an error? Otherwise, we can do without errors, e.g.,:

```py
ks.options.compute = 1
```

Maybe we should raise an error like this case.

Reply:
Yea, I was trying to find a simpler way .. let me check today.